LoopKitUI/Extensions/Environment+Colors.swift

```diff
@@ -51,3 +66,18 @@ public extension EnvironmentValues {
         set { self[InsulinTintColorKey.self] = newValue }
     }
 }
+
+private struct ChartColorPaletteKey: EnvironmentKey {
+    static let defaultValue: ChartColorPalette = ChartColorPalette(axisLine: .clear,
+                                                                   axisLabel: .secondaryLabel,
+                                                                   grid: .systemGray3,
+                                                                   glucoseTint: .systemTeal,
+                                                                   insulinTint: .orange)
```

Comment:
Should these use the default colours for glucose and insulin defined above?

Reply:
Yes, except for the problem of converting from the default's `Color` (a SwiftUI View) type to the `UIColor` type needed here. If we bumped `LoopKit` to require iOS 14 then there is an easy way to fix this (a new `UIColor` initializer than takes a `Color`), but without that it is a lot of extra, unnecessary code to convert.  I figured since this was all in the same file it wasn't that bad to not reference the default colors. 

What do you think?